tests/test_call.py

```diff
@@ -1393,3 +1393,285 @@ def test_invalid_header_pairwise_distances(tmp_path):
 
     assert exception.type == Exception
     assert str(exception.value) == f'{pairwise_distances_path} does not appear to be a properly TSV-formatted file'
+
+def test_batch_size_n1(tmp_path):
+    # batch_size = -1
+    # The (pairwise distance) input has 2 elements {E, F}
+    config = {}
+
+    clusters_path = get_path("data/clusters/basic.tsv")
+    pairwise_distances_path = get_path("data/pairwise_distances/basic.tsv")
+    output_path = path.join(tmp_path, "test_out")
+
+    config["dists"] = pairwise_distances_path
+    config["rclusters"] = clusters_path
+    config["outdir"] = output_path
+    config["force"] = False
+
+    config["thresholds"] = "5,3,0"
```

Comment:
Can we also test if thresholds are provided in wrong increasing order and are not monotonically decreasing? Say `5,0,3` instead of `5,3,0`?